include/epiworld/database-bones.hpp

```diff
@@ -263,7 +263,8 @@ class DataBase {
         std::string fn_reproductive_number,
         std::string fn_generation_time,
         std::string fn_active_cases,
-        std::string fn_outbreak_size
+        std::string fn_outbreak_size,
+        std::string fn_hospitalizations = ""
```

Comment:
We should be more consistent. This would require to modify whatever file calls this function in `examples` and `tests`

```suggestion
        std::string fn_hospitalizations
```